### DISCUSSION TITLE
помощник по персонажу

### DISCUSSION THREAD

**Discussion by Kor-Angar** [dosinabox/g2nr_unofficial_update#466] [General]
...

**Comment by Kor-Angar** [comment]
@dosinabox в алхимии среди сложных напиток нет эликсира жизни и маны

**Reply by Kor-Angar** [reply]
![20220815234452_1](https://user-images.githubusercontent.com/49514764/184733471-4ad05f89-857c-42db-b92d-9e2d308bf97f.jpg)

**Reply by Kor-Angar** [reply]
так же можно добавить "выучить все рецепты"

**Reply by dosinabox** [reply]
> @dosinabox в алхимии среди сложных напиток нет эликсира жизни и маны

Они почему-то в обычных зельях. Надо перенести в особые, а туда добавить чистое здоровье и чистую ману.

**Reply by Kor-Angar** [reply]
так и есть. про чистое здоровье и ману забыл написать :-) для себя сделал , а отписаться забыл :-( дергают меня постоянно. отвлекают.

**Reply by Kor-Angar** [reply]
>  Надо перенести в особые,

может быть даже лучше продублировать.

**Reply by Kor-Angar** [reply]
и ещё наверное добавить мегадринк и выдачу компонентов на него.

**Reply by dosinabox** [reply]
Добавлю мегадринк, потому что он есть в массиве PLAYER_TALENT_ALCHEMY и для него прописана стоимость обучения (20).
Дублировать не надо, лучше пусть структура меню обучения соответствует меню изготовления.

**Reply by Kor-Angar** [reply]
> Дублировать не надо, лучше пусть структура меню обучения соответствует меню изготовления.

а там же в НВ "вынесли" сложные напитки. и поэтому на хп и мана работает схема из классики , а на силу и ловкость схема из НВ. оттого "структура" сломалась. получается с маной и хп ни два ни полтора.

**Comment by Kor-Angar** [comment]
@dosinabox добавь возможность плиз выучить сразу 6 кругов магии.
